.github/workflows/yocto.yml

```diff
@@ -0,0 +1,55 @@
+name: Yocto devtool test
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build with Yocto devtool on Ubuntu
+    runs-on: ubuntu-20.04
```

Comment:
Will there be no job on Ubuntu 22.04?

Reply:
Yocto devtool is a build tool with isolated env; it's like "gbs" for "Yocto OS". We don't need to test with multiple Ubuntu versions.